lib/api/v3/groups/group_representer.rb

```diff
@@ -33,6 +33,7 @@ module V3
     module Groups
       class GroupRepresenter < ::API::V3::Principals::PrincipalRepresenter
         include API::Decorators::LinkedResource
+        extend ::API::V3::Utilities::CustomFieldInjector::RepresenterClass
```

Comment:
Now that groups also support custom fields in the API (such a simple change, glad that it is finally done), it would be good to briefly document that. 

On other endpoints we have something like this below the links table:

'Depending on custom fields defined for time entries, additional links might exist.'

And this below the properties list:

'Depending on custom fields defined for time entries, additional properties might exist.'

Reply:
https://github.com/opf/openproject/pull/19455/commits/2f3d9440bc24d486eda04f06c2561a93d5aefed8